tests/server_test_cases/config_tests.py

```diff
@@ -440,7 +440,7 @@ def test_analytics_id(self):
         config.set(analytics_id='analytics_id_xyz')
 
         doc = self.go('/haiti/create')
-        assert "getTracker('analytics_id_xyz')" in doc.content
+        assert "gaProperty': 'analytics_id_xyz'" in doc.content
```

Comment:
After putting the first quotation ' in resoure_tests.py, server test failed with the error below.
Am I missing something or can I ignore this?

_____________________ ResourceTests.test_resource_caching ______________________
tests/server_test_cases/resource_tests.py:92: in test_resource_caching
    assert doc.content_bytes == 'bonjour'
E   AssertionError: assert 'goodbye' == 'bonjour'
E     - goodbye
E     + bonjour
----------------------------- Captured stderr call -----------------------------
> GET http://localhost:8081/personfinder/global/foo.txt
> GET http://localhost:8081/personfinder/global/foo.txt?lang=fr
> GET http://localhost:8081/personfinder/global/foo.txt?lang=fr
> GET http://localhost:8081/personfinder/global/foo.txt?lang=fr
> GET http://localhost:8081/personfinder/global/foo.txt?lang=fr
> GET http://localhost:8081/personfinder/global/foo.txt?lang=fr
> GET http://localhost:8081/personfinder/global/foo.txt?lang=fr
1 failed, 134 passed in 477.62 seconds